examples/selfie/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import type { CustomAction, CustomEvent } from "@mirohq/websdk-types";
+
+const registerAction = async () => {
```

Comment:
Can you move this function into its own file (i.e. `register-custom-miro-action.ts`)? That will make it easier to find the custom action creation for users